include/exadg/grid/grid.cpp

```diff
@@ -132,6 +132,11 @@ Grid<dim>::create_triangulation(
   const unsigned int                                        global_refinements,
   const std::vector<unsigned int> &                         vector_local_refinements)
 {
+  if(data.element_type == ElementType::Simplex)
+    AssertThrow(vector_local_refinements.size() == 0,
+                dealii::ExcMessage(
+                  "Currently, local refinements are not supported for simplex meshes."));
```

Comment:
It is currently not clear to me why this Assert is written outside the function `do_create_triangulation()` below? All the parameters (`data` and `vector_local_refinements`) required for this code are passed to `do_create_triangulation()` if I see it correctly.

Reply:
Another question would be whether this change should be done in the present PR. Why not in #295 where you make many changes to the `Grid` class.

Reply:
let's do this in a separate PR.